### ISSUE TITLE
explanation of hparams and config.json

### ISSUE THREAD

**Issue by Ahmad-noborders** [NVIDIA/waveglow#218]
@EuphoriaCelestial do all the parameters need to be exactly same in both hparams and config.json file? can you please elaborate these params what they do and how they matter and impact training and results.
cc: @rafaelvalle 

- [ ]  fp16_run
- [ ] sigma
- [ ] learning rate
- [ ] batch size
- [ ] segment_length
- [ ] sampling_rate
- [ ] filter_length
- [ ] hop_length
- [ ] win_length
- [ ] mel_fmin
- [ ] mel_fmax

**Comment by EuphoriaCelestial** [comment]
I can only explain a few of them:
fp16_run: enable half precision mode, which increase the batch_size you can use to training -> faster training
sigma: not sure what this is, but it affect the output audio, you can change it when run inference to see the different (dont change during training process)
learning_rate: this is basic
segment_length: increase this if you increase sampling rate and vice versa
sampling_rate: you already known this

I havent done any experiment with the last 5 params, maybe you should ask someone else
